scripts/us_census/pep/pep_by_srh/process.py

```diff
@@ -43,13 +47,19 @@
 input files, creates necessary folders for processing.
```

Comment:
Pls update the download instructions as download.sh is deleted.

Reply:
Updated.